recipes/mlrose/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "mlrose" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cec83253bf6da67a7fb32b2c9ae13e9dbc6cfbcaae2aa3107993e69e9788f15e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - numpy
+    - scipy
+    - scikit-learn <0.21.0
```

Comment:
Does the package really need such an old version of `scikit-learn`?

Fine if it's really the case, but you should open an issue upstream so it's updated to support newer versions.
It will be difficult to install with other packages otherwise.

Reply:
I just found this [PR](https://github.com/gkhayes/mlrose/pull/50) and a comment that recommends to move to [mlrose-hiive](https://github.com/hiive/mlrose).

Seems like `mlrose` isn't maintained anymore.
Might be better to submit a recipe for `mlrose-hiive`?

Reply:
@beenje I also found that to be the case. I'll be updating this to use mlrose-hiive.